src/utils/actionUtils.ts

```diff
@@ -5,16 +5,43 @@
 
 import { ActionsGetJobForWorkflowRunResponseData, ActionsGetWorkflowRunResponseData } from '@octokit/types';
 import * as moment from 'moment';
-import * as path from 'path';
+import { ThemeIcon } from 'vscode';
 import { TreeItemIconPath } from 'vscode-azureextensionui';
 import { ActionWorkflowStepData, Conclusion, Status } from "../gitHubTypings";
 import { localize } from "./localize";
-import { treeUtils } from "./treeUtils";
 
 export function getActionIconPath(data: ActionWorkflowStepData | ActionsGetJobForWorkflowRunResponseData | ActionsGetWorkflowRunResponseData): TreeItemIconPath {
-    return data.conclusion !== null ?
-        treeUtils.getThemedIconPath(path.join('conclusions', ensureConclusion(data))) :
-        treeUtils.getThemedIconPath(path.join('statuses', ensureStatus(data)));
+    let id: string;
+    if (data.conclusion !== null) {
+        switch (ensureConclusion(data)) {
+            case Conclusion.Cancelled:
+                id = 'circle-slash';
+                break;
+            case Conclusion.Failure:
+                id = 'error';
+                break;
+            case Conclusion.Skipped:
+                id = 'debug-step-over';
```

Comment:
My only issue is this icon <img width="34" alt="Screen Shot 2021-04-08 at 11 39 23 AM" src="https://user-images.githubusercontent.com/22795803/114079597-3c8b2880-985f-11eb-8f1c-cb94e35b208c.png">

 It reminds me too much of liveshare lol. I do prefer the GitHub icons here but if we have to use the codicons maybe we could match them more closely to what the GitHub icons were?

Reply:
You're welcome to suggest an alternative: https://code.visualstudio.com/api/references/icons-in-labels

I chose that one based on this entry:
```
testing-skipped-icon | debug-step-over | Icon shown for tests that are skipped.
```

Reply:
How about `stop` <img width="22" alt="Screen Shot 2021-04-08 at 11 50 47 AM" src="https://user-images.githubusercontent.com/22795803/114080894-bd96ef80-9860-11eb-9515-7d09829d194e.png"> for `Conclusion.Failure` and
`circle-slash` <img width="22" alt="Screen Shot 2021-04-08 at 11 51 05 AM" src="https://user-images.githubusercontent.com/22795803/114080916-c7205780-9860-11eb-9601-178bd62978cd.png"> for `Conclusion.Skipped`?

Reply:
Pretty sure "error" is the exact same as "stop" and I'm already using "circle-slash" for "Conclusion.Cancelled"

> I do prefer the GitHub icons here

I don't disagree, but I think we have to use VS Code themes for accessibility reasons and to satisfy all users. Ultimately, you're welcome to make your own product icon theme using the existing GitHub icons to replicate the old behavior :P

Reply:
LOL is that last comment what pushed Will to just approve the PR? 😋

Reply:
![download](https://user-images.githubusercontent.com/22795803/114085087-c2aa6d80-9865-11eb-8724-1e3412273aaf.gif)